pkg/skaffold/config/util_test.go

```diff
@@ -498,11 +498,11 @@ func TestShouldDisplayPrompt(t *testing.T) {
 			},
 		},
 		{
-			description: "should display prompt when last prompted is before 2 weeks",
+			description: "should display prompt when last prompted is older than 3 months",
```

Comment:
These test descriptions should be expressed in days since they don't properly account for months with 28, 29, 30, 31 days.

Reply:
I don't understand - these are only text descriptions, and the other tests use the term 'months' as well? Should I be changing all of the existing test descriptions to say 'days' instead? This feels somewhat arbitrary.

Reply:
I believe you should be changing the test descriptions.  The test descriptions were being expressed in relation to the previous date offsets of 3 months and 10 days.  You're revising the date offsets, and it just happens that the chosen dates also work for your revised date offsets (which means these tests weren't testing boundary conditions very precisely).  Most notable is the next test "_ should display prompt when last taken is before than 3 months ago_" which from its description should fail since you've changed the last-taken period to be 365 days ago.  But it passes because the probed date was more than 365 days ago.